pages/src/benefits/benefit.ts

```diff
@@ -0,0 +1,100 @@
+import {ready} from '../../../src/modules/esl-utils/decorators/ready';
+import {memoize} from '../../../src/modules/esl-utils/decorators/memoize';
+import {bind} from '../../../src/modules/esl-utils/decorators/bind';
+import {ESLBaseElement} from '../../../src/modules/esl-base-element/core';
+
+
+const options = {
+  threshold: [0.5]
+};
+
+interface Iconfig {
+  group?: boolean;
+  repeat?: boolean;
+  delete?: boolean;
+}
+
+class AnimateService {
+
+  private _markedElements: Element[] = [];
+  private _io = new IntersectionObserver(this.onIntersect, options);
+  private _configMap = new WeakMap<Element, Iconfig>();
+
+  @memoize()
+  static get instance() {
+    return new AnimateService();
+  }
+
+  configFor(el: Element): Iconfig | undefined {
```

Comment:
It makes sense to start methods from verbs.
The name itself explains what the method does. 
I suggest `getConfigFor()`

Reply:
Not necessary to rename in case we will have a context. but in the general situation, I agree.